.github/workflows/test.yml

```diff
@@ -12,13 +12,32 @@ on:
 jobs:
   build:
     runs-on: ubuntu-22.04
-
     strategy:
       matrix:
         node-version: [16.x]
+    services:
+      postgres:
+        image: postgres:11.21-bullseye
```

Comment:
Is there any reason we perform this test on postgres 11 and not the newest postgres 15? If our goal is to make sure db2dbml work on all version from 11->15, I think we should test both